spec/lib/puppet_spec/unindent.rb

```diff
@@ -0,0 +1,5 @@
+class String
+  def unindent
```

Comment:
If someone else also monkey patches String with the same method and does something differently, then a lot of tests will fail of course, but I'm not sure I'd call that a risk.